README.md

```diff
@@ -1,178 +1,203 @@
+- **Array** `input`: The array containing arguments that will be passed to each benchmarked function. Therefore, the number of
+elements in the `input` array should match the number of arguments of
+each specified function (i.e. the array will be "unpacked" when
+invoking functions, they will not receive a single array argument).
 
-**Execute benchmarks**
+#### Return
+- **BenchTable** The `BenchTable` instance.
 
-```javascript
-Benchtable.prototype.run(config);
-```
+### Events
+
+#### `complete`
+Get results table.
 
-* `config` (object) -- [see benchmark.js API docs](http://benchmarkjs.com/docs#Suite_prototype_run)
+After all benchmarks are finished, the `complete` event is fired and the `cli-table` table instance with the results is
+available through the `BenchTable.prototype.table` property.
 
-**Get results table**
+The ASCII string serialization of the table is obtained by calling `.toString()` on that object.
 
-```javascript
-Benchtable.prototype.on('complete', function() {
-  this.table.toString();
+```js
+var suite = new BenchTable();
+...
+suite.on('complete', () => {
+  suite.table.toString();
 });
 ```
 
-* after all benchmarks are finished, the `complete` event is fired and the `cli-table` table instance with the results is available through the `Benchtable.prototype.table` property.
-The ascii string serialization of the table is obtained by calling `.toString()` on that object.
-
-## Example
+## Examples
 
 This is the Benchtable version of the example from the [Benchmark.js homepage](http://benchmarkjs.com).
-Also available in the [examples directory](https://github.com/izuzak/benchtable/examples/string_search.js).
-
-```javascript
-// import module
-var Benchtable = require('benchtable');
-
-// create benchtable suite
-var suite = new Benchtable();
-
-// add functions
-suite.addFunction('RegExp#test', function(s) { /o/.test(s); })
-.addFunction('String#indexOf', function(s) {s.indexOf('o') > -1;})
-
-// add inputs
-.addInput('Short string', ['Hello world!'])
-.addInput('Long string', ['This is a very big string, isnt it? It is. Really. So, hello world!'])
-.addInput('Very long string', ['This is a ' + new Array(200).join("very ") + 'big string, isnt it? It is. Really. So, hello world!'])
-.addInput('Extremely long string', ['This is a ' + new Array(20000).join("very ") + 'big string, isnt it? It is. Really. So, hello world!'])
-
-// add listeners
-.on('cycle', function(event) {
-  console.log(String(event.target));
-})
-.on('complete', function() {
-  console.log('Fastest is ' + this.filter('fastest').pluck('name'));
-  console.log(this.table.toString());
-})
-
-// run async
-.run({ 'async': false });
+Also available in the [examples directory](/examples/string_search.js).
+
+### Default behavior
+
+```js
+// Import the benchtable module
+var BenchTable = require('benchtable');
+
+// Create benchtable suite
+var suite = new BenchTable();
+
+suite
+  benchtable/ Add functions for benchmarking
```

Comment:
I think there's something wrong with this line -- there's a single `/` and I think there should be two because it should be a comment, right? And it should be in a different place?